assets/setup-wizard/steps.js

```diff
@@ -33,6 +33,10 @@ const steps = [
 		key: 'features',
 		container: <Features />,
 	},
-];
+].filter(
```

Comment:
It's the main part of this PR, removing the theme step if the course theme is installed. The rest is basically a refactor to make the setup wizard behave with a linear navigation.